forest/src/cli/chain_cmd.rs

```diff
@@ -72,10 +72,19 @@ pub enum ChainCommands {
         #[structopt(short, help = "Input a valid CID")]
         cid: String,
     },
+
+    /// Fetches the most recent snapshot from a trusted, pre-defined location.
+    #[structopt()]
```

Comment:
This macro doesn't need to be here. As long as the enum it belongs to derives `StructOpt`, it will still be recognized as a command

Reply:
nice catch, thanks!